src/js/base/module/Editor.js

```diff
@@ -733,13 +738,6 @@ define([
     this.empty = function () {
       context.invoke('code', dom.emptyPara);
     };
-
-    /**
-     * set height for editable
-     */
-    this.setHeight = function (height) {
-      $editable.outerHeight(height);
-    };
   };
```

Comment:
I added setHeight method because it can't set height dynamically  out of summernote